---
cve: 2019-19588
title: CVE in validators
description: >
    The validators package 0.12.2 through 0.12.5 for Python enters an infinite loop when validators.domain is called with a crafted domain string. This is fixed in 0.12.6.
cvss_v2: 7.8
references:
    - https://github.com/kvesteri/validators/issues/86
affected:
    - name: validators
      version:
        - "<=0.12.5,0.12.2"
      fixedin:
        - ">=0.12.6"
        - "<=0.12.1,0.1.0"
